src/js/experimental/media-chrome-menu-button.js

```diff
@@ -8,56 +8,18 @@ const template = document.createElement('template');
 template.innerHTML = `
   <style>
   :host {
-    display: inline-flex;
+    display: inline-block;
```

Comment:
`inline-flex` seems to be the default for control bar items, but maybe that should be overridden by the control bar itself for slotted items? Like, maybe chrome-button should default to inline-block to be overridden by the control-bar, as well